.github/workflows/release.yml

```diff
@@ -389,6 +390,26 @@ jobs:
             workspace/runtime-*.yml
             workspace/emojivoto-demo.yml
             workspace/mysql-demo.yml
+      - name: AWS login (IAM role)
+        uses: aws-actions/configure-aws-credentials@f24d7193d98baebaeacc7e2227925dd47cc267f5 # v4.2.0
+        with:
+          role-to-assume: arn:aws:iam::795746500882:role/ContrastPublicBucketRW
+          aws-region: eu-central-1
+      - name: Upload pre-release artifacts to S3 bucket contrast-public
+        run: |
+          unix=$(date +%s)
+          files=(
+            workspace/contrast-cli/bin/contrast
+            workspace/contrast-enterprise-cli/bin/contrast-enterprise
+            workspace/coordinator*.yml
+            workspace/runtime-*.yml
+            workspace/emojivoto-demo.yml
+            workspace/mysql-demo.yml
+          )
+          for file in "${files[@]}"; do
+            aws s3 cp "$file" "s3://contrast-public/pre-releases/${{ inputs.version }}/$unix/"
+          done
+          echo "Artifacts available at https://contrast-public.s3.eu-central-1.amazonaws.com/releases/${{ inputs.version }}/$unix/" | tee -a "$GITHUB_STEP_SUMMARY"
```

Comment:
I think the artifacts might be easier to consume programmatically if they were not timestamped. Having two release candidates simultaneously sounds invalid anyway, and maybe we can live with the race condition between a re-run of the release pipeline and artifacts being consumed?

Reply:
Not sure how that makes it easier. I think the team consuming this can easily pick up the URL in the summary and use it. I added this because it allows to re-release with a minor change while the previous pre-release artifacts are still under test, in situations where we are confident that this won't affect the test result.

Alternatively, I would need to ensure we delete all the existing files before uploading the new ones.

Reply:
sgtm if programmatic access is not a priority.